tests/api/v1/botData/get.js

```diff
@@ -188,6 +188,36 @@ describe('tests/api/v1/botData/get.js >', () => {
     .catch(done);
   });
 
+  it('Pass, get data by room and bot name', (done) => {
```

Comment:
Add failing test

Reply:
Just added a failing test at the bottom of the tests but it didn't clear this comment.